src/sidebar/store/modules/test/drafts-test.js

```diff
@@ -0,0 +1,228 @@
+'use strict';
+
+const immutable = require('seamless-immutable');
+
+const drafts = require('../drafts');
+const { Draft } = require('../drafts');
+const createStore = require('../../create-store');
+
+const fixtures = immutable({
+  draftWithText: {
+    isPrivate: false,
+    text: 'some text',
+    tags: [],
+  },
+  draftWithTags: {
+    isPrivate: false,
+    text: '',
+    tags: ['atag'],
+  },
+  emptyDraft: {
+    isPrivate: false,
+    text: '',
+    tags: [],
+  },
+  annotation: {
+    id: 'my_annotation',
+    $tag: 'my_annotation_tag',
+  },
+});
+
+describe('Drafts Store', () => {
+  let store;
+
+  beforeEach(() => {
+    store = createStore([drafts]);
+  });
+
+  describe('Draft', () => {
+    it('constructor', () => {
+      const draft = new Draft(fixtures.annotation, fixtures.draftWithText);
+      assert.deepEqual(draft, {
+        annotation: {
+          ...fixtures.annotation,
+        },
+        ...fixtures.draftWithText,
+      });
+    });
+    describe('#isEmpty', () => {
+      it('creates a non empty draft object', () => {
```

Comment:
```suggestion
      it('returns false if draft has tags or text', () => {
```

The test description should be written in terms of what the method being tested returns or does, preferably in a way that is easy to relate to end-user behaviour. In this case we disable certain UI elements until an annotation being edited has either some tags or text.